### PR TITLE
Cleanup legacy

### PR DESCRIPTION
**Pull request by ArneD**

### CODE REVIEW

File: src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractProjections.cs [hunk 1]
```diff
@@ -15,21 +15,21 @@ namespace ParcelRegistry.Projections.Extract.ParcelExtract
 
     [ConnectedProjectionName("Extract percelen")]
     [ConnectedProjectionDescription("Projectie die de percelen data voor het percelen extract voorziet.")]
-    public class ParcelExtractV2Projections : ConnectedProjection<ExtractContext>
+    public class ParcelExtractProjections : ConnectedProjection<ExtractContext>
```
Comment (jvandaal) [inline src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractProjections.cs:18]: @ArneD What impact does renaming the project have on the current ProjectionState?

File: src/ParcelRegistry.Projections.Legacy/LegacyContext.cs [hunk 2]
```diff
@@ -3,17 +3,13 @@ namespace ParcelRegistry.Projections.Legacy
...
-    
+
     public class LegacyContext : RunnerDbContext<LegacyContext>
     {
         public override string ProjectionStateSchema => Schema.Legacy;
         internal const string ParcelDetailV2ListCountName = "vw_ParcelDetailV2ListCount";
-
-        public DbSet<ParcelDetailV2.ParcelDetailV2> ParcelDetailV2 { get; set; }
-        public DbSet<ParcelDetailWithCount> ParcelDetailWithCountV2 { get; set; }
+        public DbSet<ParcelDetail.ParcelDetail> ParcelDetailWithCountV2 { get; set; }
```
Comment (jvandaal) [inline src/ParcelRegistry.Projections.Legacy/LegacyContext.cs:12]: Rename without V2?

File: src/ParcelRegistry.Projections.Legacy/Migrations/20221123144056_AddV2Tables_P2.cs [hunk 3]
```diff
@@ -20,7 +20,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
             WITH SCHEMABINDING
             AS
             SELECT COUNT_BIG(*) as Count
-            FROM [{Infrastructure.Schema.Legacy}].[{ParcelDetailV2.ParcelDetailV2Configuration.TableName}]
+            FROM [{Infrastructure.Schema.Legacy}].[ParcelDetailsV2]
```
Comment (jvandaal) [inline src/ParcelRegistry.Projections.Legacy/Migrations/20221123144056_AddV2Tables_P2.cs:23]: Shouldn't it be ParcelDetails? 

```
    public class ParcelDetailConfiguration : IEntityTypeConfiguration<ParcelDetail>
    {
        internal const string TableName = "ParcelDetails";
```

Comment (ArneD) [reply src/ParcelRegistry.Projections.Legacy/Migrations/20221123144056_AddV2Tables_P2.cs:23]: Not at this point. It's still ParcelDetailsV2

**Review by jvandaal — APPROVED** [review]

**Pull request merged**